xbmc/addons/AddonManager.h

```diff
@@ -331,6 +331,38 @@ namespace ADDON
 
     AddonOriginType GetAddonOriginType(const AddonPtr& addon) const;
 
+    /*!
+     * @brief Addon update and install managment.
+     *
+     * Parts inside here are used for changes about addon system.
+     *
+     * @warning This should be never used from other places outside of addon
+     * system directory.
+     *
+     * Currently listed call sources:
+     * - @ref CAddonInstallJob::DoWork
+     */
+    /*@{{{*/
```

Comment:
About this think to look for cleanup of manager to have them better sorted and easier to see for where the are.